{
	"id": "oval:org.opensuse.security:def:202452304",
	"version": "1",
	"class": "vulnerability",
	"metadata": {
		"title": "CVE-2024-52304",
		"affected": {
			"family": "unix",
			"platform": [
				"openSUSE Leap 15.6"
			]
		},
		"reference": [
			{
				"ref_id": "Mitre CVE-2024-52304",
				"ref_url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-52304",
				"source": "CVE"
			},
			{
				"ref_id": "SUSE CVE-2024-52304",
				"ref_url": "https://www.suse.com/security/cve/CVE-2024-52304",
				"source": "SUSE CVE"
			}
		],
		"description": "\n    aiohttp is an asynchronous HTTP client/server framework for asyncio and Python. Prior to version 3.10.11, the Python parser parses newlines in chunk extensions incorrectly which can lead to request smuggling vulnerabilities under certain conditions. If a pure Python version of aiohttp is installed (i.e. without the usual C extensions) or `AIOHTTP_NO_EXTENSIONS` is enabled, then an attacker may be able to execute a request smuggling attack to bypass certain firewalls or proxy protections. Version 3.10.11 fixes the issue.\n    ",
		"advisory": {
			"from": "security@suse.de",
			"severity": "Moderate",
			"cve": [
				{
					"text": "CVE-2024-52304 at SUSE",
					"href": "https://www.suse.com/security/cve/CVE-2024-52304/",
					"impact": "medium",
					"cvss3": "5.3/CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:L/A:N"
				},
				{
					"text": "CVE-2024-52304 at NVD",
					"href": "https://nvd.nist.gov/vuln/detail/CVE-2024-52304"
				}
			],
			"bugzilla": [
				{
					"text": "SUSE bug 1233447",
					"href": "https://bugzilla.suse.com/1233447"
				}
			],
			"issued": {
				"date": "2024-11-20"
			},
			"updated": {
				"date": "2024-11-20"
			},
			"affected_cpe_list": {
				"cpe": [
					"cpe:/o:opensuse:leap:15.6"
				]
			}
		}
	},
	"criteria": {
		"operator": "AND",
		"criterions": [
			{
				"test_ref": "oval:org.opensuse.security:tst:2009864054",
				"comment": "openSUSE Leap 15.6 is installed"
			},
			{
				"test_ref": "oval:org.opensuse.security:tst:2009799750",
				"comment": "python311-aiohttp is affected"
			}
		]
	}
}
